Source/Laser/LaserParticleContainer.cpp

```diff
@@ -503,8 +503,14 @@ LaserParticleContainer::Evolve (int lev,
             //
             // Current Deposition
             //
-            DepositCurrent(pti, wp, uxp, uyp, uzp, jx, jy, jz,
-                           cjx, cjy, cjz, np_current, np, thread_num, lev, dt);
+            // Deposit inside domains
+            DepositCurrent(pti, wp, uxp, uyp, uzp, &jx, &jy, &jz,
+                           0, np_current, thread_num,
+                           lev, lev, dt, jx.nGrow());
+            // Deposit in buffers
+            DepositCurrent(pti, wp, uxp, uyp, uzp, cjx, cjy, cjz,
+                           np_current, np-np_current, thread_num,
+                           lev, lev-1, dt, jx.nGrow());
```

Comment:
Should this be `cjx.nGrow()` ?

Reply:
In the previous implementation, both `jx` and `cjx` were passed as arguments, and `ngJ` was defined by `long ngJ = jx.nGrow();`, so I just kept it the same way as it was before (and this is actually why `ngJ` has to be passed as an argument here).

Reply:
But I think you are right. Let me test it and commit it.